### PR TITLE
Fix multiple flag colors and/or shape

### PR DESCRIPTION
**Pull request by wyattoday**
- Canadian flag with correct color and shape from here: https://commons.wikimedia.org/wiki/File:Flag_of_Canada_(Pantone).svg
- United Kingdom flag with correct color from here: https://commons.wikimedia.org/wiki/File:Flag_of_the_United_Kingdom.svg
- New Zealand flag colors. Source: https://commons.wikimedia.org/wiki/File:Flag_of_New_Zealand.svg
- Australian flag colors. Source: https://commons.wikimedia.org/wiki/File:Flag_of_Australia_(converted).svg
- South Korean flag colors. Source: https://commons.wikimedia.org/wiki/File:Flag_of_South_Korea.svg
- Israel flag svg size reduced.
- Iraq flag rendering. Source: https://commons.wikimedia.org/wiki/File:Flag_of_Iraq.svg
- South African flag colors. Source: https://commons.wikimedia.org/wiki/File:Flag_of_South_Africa.svg
- Puerto Rico flag rendering (smaller file, more accurate). Source: https://commons.wikimedia.org/wiki/File:Flag_of_Puerto_Rico.svg
- Belize flag color and design errors, Source: https://commons.wikimedia.org/wiki/File:Flag_of_Belize.svg
- Cyprus flag errors rendering. Source: https://commons.wikimedia.org/wiki/File:Flag_of_Cyprus.svg
- Eswatini SVG flag size reduced. Identical rendering.
- Angola flag (errors in design and colors). Source: https://commons.wikimedia.org/wiki/File:Flag_of_Angola.svg
- Namibia flag "escaping" the viewbox (added clipping path). Source: https://commons.wikimedia.org/wiki/File:Flag_of_Namibia.svg
- Flag of Spain, reduced size of SVG. Source: https://en.wikipedia.org/wiki/File:Flag_of_Spain.svg
- Update Kosovo flag to meet their specs. Source: https://commons.wikimedia.org/wiki/File:Flag_of_Kosovo.svg
- 4 more flags: Albania, Anguilla, Aruba, Bahrain

Sources: https://commons.wikimedia.org/wiki/File:Flag_of_Albania.svg , https://commons.wikimedia.org/wiki/File:Flag_of_Anguilla.svg , https://commons.wikimedia.org/wiki/File:Flag_of_Aruba.svg , https://commons.wikimedia.org/wiki/File:Flag_of_Bahrain.svg

Etc. See commits for all sources and where we occasionally veered from the source where they were incorrect (tc.svg for example).

### CODE REVIEW

**Comment by JimmyCushnie** [comment]
lmao if you're just going to keep making commits you should make this a draft PR until you're done. You don't want this to be merged when you're not finished with it

**Comment by wyattoday** [comment]
All done. There are likely other flags that have or need fixes. For example, the American Samoa SVG included in this PR -- pulled from Wiki Commons -- looks closer to the actual flag, but unfortunately not exactly like it.

So, there's definitely work to be done both in this repository and on the wider internet to get accurate designs (color, layout, size, etc.).

Some regions accurately describe (often through legislation) how the flag looks. Others seemingly don't (or at least don't do it in a way accessible from the internet).

But this PR brings things closer to how things are in reality. So I'd recommend merging this.

**Comment by wyattoday** [comment]
Tell me your thoughts, @hjnilsson , but I think this repository needs needs a few big changes:

1. Automated scripts to "check-in" on wiki commons changes to all of the flags. These changes shouldn't be auto-merged, but new issues should be created to monitor the changes and/or adopt them. Some changes are stupid turf wars over "Wiki reputation" and other nonsense, so some changes should be outright ignored.

2. Links to legislation or reference drawing defining (accurately as possible) the shape and layout of flags.

#1 is a bit of work, but it will solve the "live" nature of country borders, flags, independence, etc. For example, Washington D.C. statehood is a very real possibility which would change the U.S. flag. Also, there are still significant "liberties" taken with other flag representations  -- i.e. they don't match the actual real flags used.

#2 is a significant project that will take many months to do.

In short, this will solve the "static" nature of this repository (it seems stuck in 2015 / 2016) and ensure the flags are as accurate as possible.

I can either do this work in the fork or do this work in this repository. I would need permissions to make commits to this repository for me to do the work here. I'm not willing to wait weeks / months to get things merged (not a knock against you, just a personal preference).

**Comment by JimmyCushnie** [comment]
@wyattoday I really appreciate all the work you've done here with flag quality control, I've been using your work in my project. Thank you. I would support and contribute to a fork or a new repo from you that does the things you're describing.

**Comment by wyattoday** [comment]
Thanks, @JimmyCushnie . I'll wait a few days before I make any moves.

**Comment by hjnilsson** [comment]
Amazing improvements! I agree on your points, and have added you as a maintainer as well. Just had a kid so a bit distracted on the maintenance front. :)

Changes looks good, great work! I'll merge these now, but we can continue discussion,

**Comment by JimmyCushnie** [comment]
Congrats on the kid!!

**Comment by wyattoday** [comment]
Congrats on the baby! They do have a way of monopolizing what was once "free" time. 😀
